app/models/container.rb

```diff
@@ -57,6 +57,8 @@ class Container < ApplicationRecord
   # Setup additional methods
   #
   def self.create_with_source(cluster_id, params)
+    param_bootstrappers = params.dig(:bootstrappers)
+    bootstrappers = JSON.parse(param_bootstrappers) if param_bootstrappers.present?
```

Comment:
why do we need to parse? the type is json right?

Reply:
We undo this changes related to this commit https://github.com/pathfinder-cm/pathfinder-mono/pull/30/commits/37d014568fb4e923f66b72e67a77ba0263904b3e
Can you please check?